SWIPs/swip-message_to_honey.md

```diff
@@ -48,19 +53,19 @@ This section describes the interaction between the nodes and the oracle in more
-No alternative solutions to updating the *relative* message pricing was considered. Noteworthy is perhaps to look at how op-code prices (in gas) are updated in Ethereum. Also here, it concerns relative prices and the developers are tweaking the gas-prices almost every hard-fork (see [opcode re-price discussion](https://ethereum-magicians.org/t/opcode-repricing/3024)). Using an oracle, updatable by a multi-signature of Swarm developers effectively also allows the developers to set prices (just as they do with ether opcodes). However, the ethereum opcode upgrade mechanism must also be approved by the majority of the hashing power of the Ethereum network (to have a succesfull hard-fork). This check is missing in the current design for `messageToHoney` price oracles. 
+No alternative solutions to updating the *relative* message pricing was considered. It is perhaps worthy to look at how op-code prices (in gas) are updated in Ethereum. Here we also have relative prices and the developers are tweaking the gas-prices on almost every hard-fork (see [opcode re-price discussion](https://ethereum-magicians.org/t/opcode-repricing/3024)). Using an oracle, updatable by a multi-signature of Swarm developers, effectively also allows the developers to set prices (just as in ethereum with ether opcodes). However, the ethereum opcode upgrade mechanism must also be approved by the majority of the hashing power of the Ethereum network (to have a successful hard-fork). This check is missing in the current design for `MsgToHoney` price oracles. 
 
 ## Backwards Compatibility 
 <!--All SWIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The SWIP must explain how the author proposes to deal with these incompatibilities. SWIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
-This SWIP is backward compatible as long as the price oracle quotes the same prices as listed in the price matrix of the non-upgraded Swarm nodes. It is up to the owner of the MsgToHoney oracle to ensure that he does not update prices too much while not all nodes run are on the new Swarm. Currently, the Swarm is not running with a live test net for settling prices, so we expect no problems with this SWIP if it is implemented before the Swarm will go live with price-incentivization. 
+This SWIP is backward compatible as long as the price oracle quotes the same prices as listed in the price matrix of the non-upgraded Swarm nodes. It is up to the owner of the `MsgToHoney` oracle to ensure that prices are not updated too much while not all running nodes are on the new Swarm. Currently, Swarm is not running a Swap-enabled live test net, so we don't expect any problems with this SWIP if it is implemented before Swarm will go live with incentivization. 
```

Comment:
Same as with other SWIP, sentence "...that prices are not updated too much while not all running nodes are on the new Swarm." is not clear

Reply:
In that case, it should be rewritten and I agree with you that it is not very well phrased here.

Reply:
I thought my PR changed this bit. It should at least say new version of Swarm code/program.